lib/util/exec/index.ts

```diff
@@ -101,11 +109,24 @@ export async function exec(
   const useDocker = binarySource === 'docker' && docker;
   if (useDocker) {
     logger.debug('Using docker to execute');
-    const dockerOptions = {
-      ...docker,
-      cwd,
-      envVars: dockerEnvVars(extraEnv, childEnv),
-    };
+    const envVars = dockerEnvVars(extraEnv, childEnv);
+    const dockerOptions = { ...docker, cwd, envVars };
+
+    if (cacheDir) {
+      const cacheVolumePrefix = dockerChildPrefix || 'renovate_';
+      const cacheVolume = `${cacheVolumePrefix}manager_cache`;
+      const cacheVolumeRoot = `/home/ubuntu`;
```

Comment:
Here we need to point some directory with permissions to write into (i.e. `user=ubuntu`)

Reply:
@viceice @rarkins Do we need to create some dedicated directory in main `Dockerfile`? Usage of the hardcoded `/home/ubuntu` doesn't seem right to me as we may shadow other important files/directories. Also we can't just mount to `/home/ubuntu/{cache}`: it will be owned by **root**.

Reply:
`/home/ubuntu` won't exist in newer renovate docker images, maybe use `process.env.HOME` ?

Reply:
So that it will assume Renovate process has same home directory as `renovate/go`, `renovate/ruby`, etc?

Reply:
🤔 Maybe better use a subdir in `baseDir` by default or `cacheDir` if defined by admin?

Yes renovate `HOME` can be different to sidecar `HOME`, as we transition to `containerbase` images, which have `HOME=/home/user`.

Reply:
Can we create some directory with user permissions, e.g. `/cache` or `/var/cache/renovate` or maybe `chown 1000 /mnt`? Something like this in `buildpack` before we release this PR?

Reply:
Okay, everything mounted inside `/tmp` directory worked fine and didn't require knowledge of user name/home.